src/sentry/interfaces/contexts.py

```diff
@@ -152,6 +152,15 @@ def __init__(self, alias, data):
         super(OsContextType, self).__init__(alias, data)
 
 
+@contexttype
+class GpuContextType(ContextType):
+    type = 'gpu'
+    indexed_fields = {
+        'name': u'{name}',
```

Comment:
Also add this since all Context have this:
```suggestion
        '': u'{name} {vender}',
```